README.md

```diff
@@ -13,6 +13,8 @@ and a [new middleware stack](https://aws.amazon.com/blogs/developer/middleware-s
 
 For more details, visit blog post on [general availability of Modular AWS SDK for JavaScript](https://aws.amazon.com/blogs/developer/modular-aws-sdk-for-javascript-is-now-generally-available/).
 
+For full [API reference please visit the documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
```

Comment:
Add further reading section from general availability blog post.
```suggestion
For more details, visit blog post on [general availability of Modular AWS SDK for JavaScript](https://aws.amazon.com/blogs/developer/modular-aws-sdk-for-javascript-is-now-generally-available/).

To get started with JavaScript SDK version 3, visit our [Developer Guide]
(https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/welcome.html)
or [API Reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/).

You can also visit our [self-guided workshop](https://github.com/aws-samples/aws-sdk-js-v3-workshop),
which builds a simple note taking application using JavaScript SDK version 2
and provides step-by-step migration instructions to version 3.

To test your universal JavaScript code in Node.js, browser and react-native environments,
visit our [code samples repo](https://github.com/aws-samples/aws-sdk-js-tests).
```

Reply:
This is great!